charts/ccd-data-store-api/values.yaml

```diff
@@ -7,8 +7,9 @@ java:
   applicationPort: 4452
   aadIdentityName: ccd
   autoscaling:
-    enabled: true 
-    maxReplicas: 8
+    enabled: true
+    maxReplicas: 12
```

Comment:
Apologies, on second thoughts.. a value of 12 would bring us over the max_connections limit in prod under full load, so we should be careful. See DATA_STORE_DB_MAX_POOL_SIZE further down in this file.